app/src/views/private/private-view.vue

```diff
@@ -421,7 +421,7 @@ function getWidth(input: unknown, fallback: number): number {
 				--v-divider-color: var(--theme--navigation--list--divider--border-color);
 				--v-divider-thickness: var(--theme--navigation--list--divider--border-width);
 
-				height: calc(100% - 64px);
+				height: calc(100% - 60px);
```

Comment:
Just a small suggestion:

For such cases I like using vars as a way to describe values. A comment would also do it. This helps to understand the value if we'd come across or have to touch it again in the future.

```suggestion
				--project-header-height: 60px;

				height: calc(100% - var(--project-header-height));
```

Reply:
added a commit